tools/release-ci/cmd/sidecar.go

```diff
@@ -55,7 +61,7 @@ func runSidecar(_ []string) error {
 		return err
 	}
 
-	if len(config.GceCredentialsFile) == 0 {
+	if len(config.GceCredentialsFile) == 0 || !exists(config.GceCredentialsFile) {
```

Comment:
Seems like specifying a file that doesn't exist is an error, not something to eat and continue